authorizerd.go

```diff
@@ -364,6 +360,10 @@ func (a *authorizer) verify(ctx context.Context, m mode, tok, act, res string, c
 
 	switch m {
 	case roleToken:
+		if isCacheHit {
```

Comment:
I' m doing this because even if I functionalize it, still need to write an IF statement.